app/models/manageiq/providers/container_manager/container_template.rb

```diff
@@ -0,0 +1,67 @@
+class ManageIQ::Providers::ContainerManager::ContainerTemplate < ContainerTemplate
```

Comment:
Specifically templates are an Openshift addition, they don't exist in Kubernetes.
(they might in future, openshift concepts tend to get upstreamed, there is already [a proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/templates.md))